test/cases/enum.ts

```diff
@@ -6,7 +6,8 @@ export var schema = {
       "enum": ["a", "b", "c"]
     },
     "bar": {
-      "enum": [1, 2, 3]
+      "enum": [1, 2, 3],
+      "tsEnumNames": ["One","Two","Three"]
```

Comment:
> I also think the values of enum should obey the "type" property in the schema (with a default of string) instead of us trying to infer what they ought to be based on context.

Agreed.

> What's the resolution here? Are we going to include case 2 + the generated util classes, or are we going to omit it?

Let's omit for now, and tackle in a separate PR. We should spec out runtime utils more before implementing them.